client/src/scenes/Battle.ts

```diff
@@ -96,6 +98,10 @@ export default class Battle extends Phaser.Scene {
         }
     }
 
+    private addRoundText() {
+        // this.roundText = this.add.text(300, 200, 'Round ' + this.room.state.currentRound, { fontSize: '30px' }).setScrollFactor(0);
```

Comment:
not too sure whats the purpose of addRoundText

Reply:
this is for future implementation, i will make use of schema field directly.